client/settings/payment-settings/account-details-section.js

```diff
@@ -113,7 +130,19 @@ const AccountDetailsSection = ( { setModalType, setKeepModalContent } ) => {
 				block: 'start',
 			} );
 		}
-	}, [ headingRef ] );
+	}, [ headingRef, isTestMode ] );
+
+	useEffect( () => {
+		const { track } = getQuery();
+		if ( track ) {
+			trackReconnectEvent( 'admin_notice', isTestMode ? 'test' : 'live' );
+
+			// Remove track param from URL.
+			const url = new URL( window.location.href );
+			url.searchParams.delete( 'track' );
+			window.location.replace( url.href );
```

Comment:
`window.location.replace( url.href );` causes a page reload, so clicking the`Reconnect to Stripe` button in the notice causes two consecutive reloads.

We should use `replaceState`:
```suggestion
			history.replaceState( null, '', url.href );
```